ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/ListFunctionsExecutor.java

```diff
@@ -48,6 +48,14 @@ private ListFunctionsExecutor() { }
             FunctionType.scalar))
         .collect(Collectors.toList());
 
+    all.addAll(functionRegistry.listTableFunctions().stream()
```

Comment:
(not your code), but instead of collecting to list at the end we should just `.forEach(all::add)`

Reply:
changed